### PR TITLE
Update Azure info

### PR DESCRIPTION
**Pull request by esell**
This PR cleans up some of the Azure information, possibly closing issue #104. It also removes the blurb about GCE being the next closest competitor to AWS as the data that is linked to in the report does not support that statement.

Disclaimer: I'm a Microsoft employee :)

### CODE REVIEW

**Comment by heneryville** [comment]
The original also several assertions that Azure is for Microsoft-first shops. Is there any data to alter those assertions?

**Comment by jlevy** [comment]
@esell thanks for the helpful comments and additions! (And also thanks for noting your affiliation!)

Definitely agree, we haven't put much effort/thought into covering Azure vs AWS in a thorough way. Also the Gartner report in the issue is a great possible addition.

Given a lot of people are reading this now, and changes wrt vendors etc are sometimes a little controversial, we're considering at least marking ones like this with a label ("neutrality") and making sure we're thoughtful about the additions and also give a short period where people can comment.

**Comment by jlevy** [comment]
I'm good with this change with two small changes:

- The line "the de facto choice for companies and teams that are focused on a Microsoft stack" is true — I don't think it should be deleted.
- If you're up for it, would you also drop in a link to the Gartner report you have in #104 with a statement like "A good overview of the major cloud players are in _this Gartner report_". I think it's very helpful.

@ThanosBaskous and others, any further thoughts?

File: README.md [hunk 1]
```diff
@@ -138,8 +138,8 @@ General Information
...
-	-	[**Microsoft Azure**](https://azure.microsoft.com/en) is the de facto choice for companies and teams that are focused on a Microsoft stack.
+	-	[**Google Cloud**](https://cloud.google.com/). It arrived later to market than AWS, but has vast resources and is now used widely by many companies, including a few large ones. It is gaining market share. Not all AWS services have similar or analogous services in Google Cloud. And vice versa: In particular Google offers some more advanced machine learning-based services like the [Vision](https://cloud.google.com/vision/), [Speech](https://cloud.google.com/speech/), and [Natural Language](https://cloud.google.com/natural-language/) APIs. It’s not common to switch once you’re up and running, but it does happen: [Spotify migrated](http://www.wsj.com/articles/google-cloud-lures-amazon-web-services-customer-spotify-1456270951) from AWS to Google Cloud. There is more discussion [on Quora](https://www.quora.com/What-are-the-reasons-to-choose-AWS-over-Google-Cloud-or-vice-versa-for-a-high-traffic-web-application) about relative benefits.
```
Comment (jlevy) [inline README.md]: Let's change this to:

[**Microsoft Azure**](https://azure.microsoft.com/en) is the de facto choice for companies and teams that are focused on a Microsoft stack, and it has now placed significant emphasis on Linux as well.

Fair enough?

**Comment by ThanosBaskous** [comment]
@jlevy I am also fine with this change, pending your requested modifications. I also agree that a short embargo of changes that involve vendors and cloud provider comparisons is a good idea (in order to allow more of a chance for the community to comment).

**Comment by esell** [comment]
ok @jlevy, added requested changes and got my fork current :)

**Comment by ThanosBaskous** [comment]
👍 lgtm @jlevy

**Comment by jlevy** [comment]
Thanks @esell for the work!

**Pull request merged**